docs/source/guides/links.rst

```diff
@@ -0,0 +1,89 @@
+Links
+~~~~~
+
+In order to write neural networks, we have to combine functions with *parameters* and optimize the parameters.
+You can use **links** to do this.
```

Comment:
I think we should use use the proper class names (in this case by capitalizing the first letter _l_ to _L_).

Reply:
Or should they be hyperlinks to the class docs?

Reply:
Continue reading I noticed this isn't so obvious. It depends on how you relate the `Link` class to the English noun _link_? Maybe this has been discussed already?